### DISCUSSION TITLE
- WARNING {neo-tree.nvim}: unknown key <deactivate>

### DISCUSSION THREAD

**Discussion by jby** [LazyVim/LazyVim#319] [Q&A] [answered]
I've started to get this warning in my healthcheck:

<img width="472" alt="image" src="https://user-images.githubusercontent.com/1584888/220416519-5676fa6d-69db-4e61-9e57-b048fffdc3ca.png">

What causes it, is it something I should worry about, can **I** do something to resolve it?

**Comment by dpetka2001** [comment, marked as answer]
As far as I know it's a new feature of `lazy.nvim` that will allow users to reload plugins without having to restart Neovim, I think. But maintainer last said that he still needs to do some testing to ensure it's working correctly, so although it is in the code it's not documented anywhere concrete yet. This [line](https://github.com/LazyVim/LazyVim/blob/db9a7d90d5d00088cfbe1236035b85299b97b885/lua/lazyvim/plugins/editor.lua#L27) here is what causes the warning, but it's nothing to worry about.

**Reply by jby** [reply]
Ok, thanks. I must have been completely useless with my grep-ing since I was unable to find the reference.

**Reply by jby** [reply]
Or not - I don't have the editor.lua file in my config... ???

**Reply by jby** [reply]
Ah, not in the config, in ~/.local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins/editor.lua

**Reply by dpetka2001** [reply]
Yes, Lazyvim's default plugins are configured in `~/.local/share/nvim/lazy/LazyVim/lua/lazyvim/plugins` and your plugins configurations (or changes to the default plugins) go into `~/.config/nvim/lua/plugins`.

**Reply by folke** [reply]
Yep, still need to add `deactivate` to the whitelist for checkhealth. But indeed nothing to worry about
